compiler+runtime/src/cpp/jank/util/escape.cpp

```diff
@@ -69,7 +69,7 @@ namespace jank::util
 
   native_persistent_string escape(native_persistent_string const &input)
   {
-    /* We can expect on relocation, since escaping anything will result in a larger string.
+    /* We can expect on relocation, since escaping results in an equal or larger length string.
```

Comment:
I'm still mulling over this comment update. Getting confused on which direction is {un}escape.

I think the original comment still holds.

Reply:
I added documentation that helped me understand the difference between unescape and escape.